Sources/docc/DocCDocumentation.docc/documenting-api-with-different-language-representations.md

```diff
@@ -0,0 +1,61 @@
+
+### Linking to Symbols with Different Language Representations
+
+You can use either source language's spelling of the symbol path to refer to the symbol in a symbol link. 
+For example, consider a `Sloth` class with `@objc` attributes:
+
+```swift
+@objc(TLASloth) public class Sloth: NSObject {
+    @objc public init(name: String, color: Color, power: Power) {
+        self.name = name
+        self.color = color
+        self.power = power
+    }
+}
+```
+
+Both ` ``Sloth/init(name:color:power:)`` ` and ` ``TLASloth/initWithName:color:power:`` ` refer to the same Sloth initializer. 
+
+Regardless of which source language's spelling you use in the symbol link, DocC matches the on-page link text to the symbol name in the source language version of the page that the reader selected. If the symbol doesn't have a representation in the source language that the reader selected, the link text will use the symbol name in the language that declared the symbol.
+
+For more information about linking to symbols, see <doc:linking-to-symbols-and-other-content>.
+
+### Document Language Specific Parameters and Return Values
+
+When a symbol has different parameters or return values in different source language representations, DocC hides the documentation for the parameters or return values that don't apply to the source language version of the page that the reader selected. For example, consider an Objective-C method with an `error` parameter and a `BOOL` return value that correspond to a throwing function without a return value in Swift:
+
+**Objective-C definition**
+
+```objc
+/// - Parameters:
+///   - someValue: Some description of this parameter.
+///   - error: On input, a pointer to an error object. If an error occurs, this pointer is set to an object containing the information about the error.
+/// - Returns: `YES` if "doing something" was successful, or `NO` if an error occurred.
+- (BOOL)doSomethingWith:(NSInteger)someValue
+                  error:(NSError **)error;
+```
+
+**Generated Swift interface**
+
+```swift
+func doSomething(with someValue: Int) throws
+```
+
+Because the Swift representation of this method only has the "someValue" parameter and no return value, DocC hides the "error" parameter documentation and the return value documentation from the Swift version of this symbol's page.
+
+
+You don't need to document the Objective-C representation's "error" parameter or return value for symbols defined in Swift.
+DocC automatically adds a generic description for the "error" parameter and extends your return value documentation to describe the Objective-C specific return value behavior. 
+
+If you want to customize this documentation you can manually document the "error" parameter and return value. 
+Doing so won't change the Swift version of that symbol's page.
+DocC will still hide the parameter and return value documentation that doesn't apply to each source language's version of that symbol's page.
```

Comment:
```suggestion
You don't need to document the Objective-C representation's `error` parameter or return value for symbols defined in Swift. DocC automatically adds a generic description for it, and extends your return value documentation to describe the Objective-C specific return value behavior. 

If you want to customize this documentation you can manually document the `error` parameter and return value. 
Doing so won't change the Swift version of that symbol's page.
DocC will still hide the parameter and return value documentation that doesn't apply to each source language's version of that symbol's page.
```

Reply:
Using "it" after a sentence that references two things (the error parameter and the return value) leads to some ambiguity regarding which of the two "it" is referring to. The second half of that sentence references the return value by name which allows the reader to deduce that "it" referred to the error parameter but I find that the paragraph is clearer when they're both explicitly spelled out.